src/Match.js

```diff
@@ -25,13 +25,20 @@ import {
   EVENT_VAR,
   MATCH_STATUS_FINISHED,
   MATCH_STATUS_LIVE,
+  SHOOT,
+  SHOOT_SAVED,
+  FOOL,
+  CORNER_SHOT,
+  OFF_SIDE,
```

Comment:
Pour des question d'homogénéité, les constantes qui contiennent la valeur d'un type doivent être préfixé par `EVENT_`